doc/connectivity/bluetooth/api/mesh/shell.rst

```diff
@@ -1296,57 +1314,58 @@ The DFU metadata commands allow generating metadata that can be used by a Target
 
 	Clear the stored composition data to be used for the Target node.
 
-``mesh models dfu metadata comp-add <cid> <pid> <vid> <crpl> <features>``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+``mesh models dfu metadata comp-add <CompanyID> <ProductID> <VendorID> <Crpl> <Features>``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 	Create a header of the Composition Data Page 0.
 
-	* ``cid``: Company identifier assigned by Bluetooth SIG.
-	* ``pid``: Vendor-assigned product identifier.
-	* ``vid``: Vendor-assigned version identifier.
-	* ``crpl``: The size of the replay protection list.
-	* ``features``: Features supported by the node in bit field format:
+	* ``CompanyID``: Company identifier assigned by Bluetooth SIG.
+	* ``ProductID``: Vendor-assigned product identifier.
+	* ``VendorID``: Vendor-assigned version identifier.
+	* ``Crpl``: The size of the replay protection list.
+	* ``Features``: Features supported by the node in bit field format:
+
 		* ``0``: Relay.
 		* ``1``: Proxy.
 		* ``2``: Friend.
 		* ``3``: Low Power.
 
-``mesh models dfu metadata comp-elem-add <loc> <nums> <numv> {<sig model id>|<vnd company id> <vnd model id>}...``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+``mesh models dfu metadata comp-elem-add <Loc> <SIGModelCount> <VndModelCount> {<SigModelId>|<VndCompanyID> <VndModelID>}...``
```

Comment:
```suggestion
``mesh models dfu metadata comp-elem-add <Loc> <SIGModelCnt> <VndModelCnt> {<SigModelID>|<VndCompanyID> <VndModelID>}...``
```
Though previous variant could be OK as well (`NumS`, `NumV`) as it is a bit shorter.

Reply:
but why `SIGModelCount` and `SigModelID`? Should it be `SIG..` or `Sig..` in both?